test/decl/overload.swift

```diff
@@ -227,7 +227,7 @@ func throwsFuncParam(_ fn: () throws -> ()) { }
 func throwsFuncParam(_ fn: () -> ()) { }
 
 // @noescape
-func noescape(x: @noescape (Int) -> Int) { } // expected-note{{previously declared}}
+func noescape(x: (Int) -> Int) { } // expected-note{{previously declared}}
 func noescape(x: (Int) -> Int) { } // expected-error{{invalid redeclaration of 'noescape(x:)'}}
```

Comment:
The goal of this test is to ensure that you cannot overload on @escaping, so you should update it. Rename noescape() to escaping(), and make one of the versions put @escaping on x